test/supportShim.js

```diff
@@ -14,6 +15,10 @@ module.exports = function(Sequelize) {
   shimAll(Sequelize.Model.prototype);
   shimAll(Sequelize.Instance.prototype);
   shimAll(QueryInterface.prototype);
+  shimAll(Sequelize.Association.prototype);
+   _.forIn(Sequelize.Association, function(Association) {
```

Comment:
An extra space has crept in at start of this line. And after all the other ones you carefully removed!